tests/hw-accel/kmm/internal/reboot/reboot.go

```diff
@@ -0,0 +1,115 @@
+package do
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/rh-ecosystem-edge/eco-goinfra/pkg/clients"
+	"github.com/rh-ecosystem-edge/eco-goinfra/pkg/nodes"
+	"github.com/rh-ecosystem-edge/eco-gotests/tests/hw-accel/kmm/internal/await"
+	"github.com/rh-ecosystem-edge/eco-gotests/tests/hw-accel/kmm/internal/kmmparams"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+)
+
+// Reboot reboots a node using the existing kmm-test-helper pod.
+func Reboot(
```

Comment:
Let's be consistent with the rest of the functions 
```suggestion
func Reboot(apiClient *clients.Settings, nodeName string, namespace string) error {
```

Reply:
done
// Reboot reboots a node using the existing kmm-test-helper pod.
func Reboot(apiClient *clients.Settings, nodeName, namespace string) error {